app/portainer/components/datatables/access-tokens-datatable/accessTokensDatatableController.js

```diff
@@ -0,0 +1,35 @@
+angular.module('portainer.app').controller('AccessTokensDatatableController', AccessTokensDatatableController);
+
+/* @ngInject */
+function AccessTokensDatatableController($scope, $controller, Authentication, DatatableService) {
```

Comment:
should use default class export instead:
```suggestion
export default class AccessTokensDatatableController {
  /* @ngInject*/
  constructor($scope, $controller, Authentication, DatatableService) {
```